shipyard-controller/Dockerfile

```diff
@@ -1,6 +1,6 @@
 # Use the official Golang image to create a build artifact.
 # This is based on Debian and sets the GOPATH to /go.
-FROM golang:1.13.7-alpine as builder
+FROM golang:1.16.3-alpine as builder
```

Comment:
Why this udpate?

Reply:
we needed to upgrade because one of the dependencies of https://github.com/benbjohnson/clock requires at least 1.15

Reply:
Thanks Florian! We should not have multiple Go versions across our services, so we should specify which one we want to have (for the next couple of months).
We have a separate issue for updating all our services to Go 1.15 (1.16 respectively): https://github.com/keptn/keptn/issues/2936 - @warber we need to up the priority on that, and switch to 1.15 or 1.16 for all services.